docs/fluvio/cli/fluvio/hub/list.mdx

```diff
@@ -18,7 +18,7 @@ Listing Hub SmartModule from the CLI is straight forward:
 $ fluvio hub list
 
   SMARTMODULE                  Visibility
-  infinyon/jolt@0.1.0          public
+  infinyon/jolt@0.4.1          public
```

Comment:
I think we can use the embed as from here:

https://github.com/infinyon/fluvio-docs/blob/main/docs/fluvio/cli/fluvio/hub/list.mdx

Reply:
sure! fixed rebasing your PR